mypy/checkmember.py

```diff
@@ -249,6 +249,20 @@ def analyze_member_var_access(name: str, itype: Instance, info: TypeInfo,
         v = Var(name, type=type_object_type(vv, builtin_type))
         v.info = info
 
+    if isinstance(vv, TypeAlias) and isinstance(vv.target, Instance):
+        # Similar to the above TypeInfo case, we allow using
+        # qualified type aliases in runtime context if it refers to an
+        # instance type. For example:
+        #     class C:
+        #         A = List[int]
+        #     x = C.A() <- this is OK
+        # As usual, we first erase any unbound type variables to Any.
+        target = set_any_tvars(vv.target, vv.alias_tvars, node.line, node.column)
+        assert isinstance(target, Instance)
+        tp = type_object_type(target.type, builtin_type)
```

Comment:
It is not exactly the same, since in one case it is `assert isinstance(...)` while in other case it is `if isinstance(...): ...`. If I exclude this, there are only two shared lines of code. So defining and importing a function will only add complexity.